adapters/adot/adot_test.go

```diff
@@ -0,0 +1,125 @@
+package adot
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+var jsonBidReq = getJsonByteForTesting("./static/adapter/adot/parallax_request_test.json")
```

Comment:
This path is already scoped to this adapter package. Why the need for so many directory levels? It's also likely not necessary as you should be able to use our standard json test framework, no?